src/components/links.js

```diff
@@ -30,6 +30,11 @@ function Links(props) {
       </div>
 
       <div className="link fadeInUp" style={{animationDelay: '0.6s'}}>
+        <h3>Crowdsourced list of Resources & Essentials from across India</h3>
+        <a href="bit.ly/covid19resourcelist">bit.ly/covid19resourcelist</a>
```

Comment:
Please add `https://` and `target="_blank"`
`<a href="bit.ly/covid19resourcelist">` ->  `<a href="https://bit.ly/covid19resourcelist" target="_blank">`